src/components/pane/pane.test.js

```diff
@@ -41,39 +40,28 @@ describe('<Pane />', () => {
         .find('li');
     });
 
-    it('should render <div> as container', () => {
-      expect(wrapper.type()).to.equal('div');
+    test('should render <div> as container', () => {
+      expect(wrapper.type()).toBe('div');
     });
 
-    it('should have the class pane', () => {
-      expect(wrapper.hasClass(classes.pane)).to.equal(true);
+    test('should have the class pane', () => {
+      expect(wrapper.hasClass(classes.pane)).toBe(true);
     });
 
-    it('should render children', () => {
-      expect(wrapper.children().length).to.equal(2);
+    test('should render children', () => {
+      expect(wrapper.children()).toHaveLength(2);
     });
 
-    it('should render tabs', () => {
-      expect(renderedTabs.length).to.equal(3);
+    test('should render tabs', () => {
+      expect(renderedTabs).toHaveLength(3);
     });
 
-    it('should have first tab as active', () => {
-      expect(renderedTabs.at(0).hasClass(classes.active));
+    test('should have first tab as active', () => {
+      expect(renderedTabs.at(0).hasClass(classes.active)).toBe(true);
     });
 
-    it('default size should be md', () => {
-      expect(renderedTabs.at(0).hasClass(classes.md)).to.equal(true);
-    });
-
-    it('should change active tab on activeTab prop change', () => {
-      wrapper.setProps({ activeTab: 1 });
-      expect(renderedTabs.at(1).hasClass(classes.active));
```

Comment:
The eslint jest plugin caught that these two never had any assertions. Currently they both fail, if I make the assumption that the author of the test wanted this to be `true`. Since this is a deprecated component, I decided not to fix these two.